### ISSUE TITLE
Accepting more than one value for more than one argument causes unexpected result

### ISSUE THREAD

**Issue by Eliran-Turgeman** [docopt/docopt#484]
I have the following docopt `__doc__` :
```
"""gene_sense_distribution_to_csv
Usage:
    gene_sense_distribution_to_csv.py <gene> <csv_out> <filenames>... [--min_count=<kn>] [--gene_regions=<kn>]
    gene_sense_distribution_to_csv.py -h | --help
    gene_sense_distribution_to_csv.py params
    gene_sense_distribution_to_csv.py example

Options:
    -h --help            Shows this screen.
    --min_count=<kn>     Minimal number of reads in a pileup line in order to consider it in the analysis [default: 0]
    --gene_regions=<kn>  All the gene regions you want to plot. Options: 5utr, 3utr, exon. The default is all of them. Separate them by space [default: ['5utr', '3utr', 'exon']]
"""
```

And i'm trying to make `--gene_regions` argument accept more than one argument like `<filenames>` does.

I tried to do that by changing this line
`gene_sense_distribution_to_csv.py <gene> <csv_out> <filenames>... [--min_count=<kn>] [--gene_regions=<kn>]`
To :
`gene_sense_distribution_to_csv.py <gene> <csv_out> <filenames>... [--min_count=<kn>] [--gene_regions=<kn>...]`

But by trying to execute the command `python SCRIPT_NAME GENE OUTPUT_PATH INPUT_PATH --gene_regions exon intron`
I get the following args:

```
{'--gene_regions': ['exon'],
 '--help': False,
 '--min_count': '0',
 '<csv_out>': 'out',
 '<filenames>': ['example', 'intron'],
 '<gene>': 'Y74C9A.6',
 'example': False,
 'params': False}
```
As you can tell, the `<filenames>` argument got the intron while I meant that `--gene_regions` would have it.

Is what i'm trying to do currently supported? and if not can it be achieved in any other way?

[stackoverflow post - if you want to answer there also](https://stackoverflow.com/questions/63557893/accepting-more-than-one-value-for-more-than-one-argument-causes-unexpected-resul)

**Comment by TylerTemp** [comment]
No. This is not supported by `docopt` because of the `docopt` parse tree. Also this is not standrad POSIX cli, because an option flag should only accept none or one arguments.

There is [another lib](http://docpie.comes.today/try?argvnofilestr=gene%20csv_out%20f1%20f2%20f3%20--gene_regions%20g1%20g2%20g3%20g4&attachopt=on&attachvalue=on&auto2dashes=on&doc=Usage%3A%5Cn%20%20%20%20gene_sense_distribution_to_csv.py%20%3Cgene%3E%20%3Ccsv_out%3E%20%3Cfilenames%3E...%20%5B--min_count%3D%3Ckn%3E%5D%20%5B--gene_regions%3D%3Ckn%3E...%5D%5Cn%20%20%20%20gene_sense_distribution_to_csv.py%20-h%20%7C%20--help%5Cn%20%20%20%20gene_sense_distribution_to_csv.py%20params%5Cn%20%20%20%20gene_sense_distribution_to_csv.py%20example%5Cn%5CnOptions%3A%5Cn%20%20%20%20-h%20--help%20%20%20%20%20%20%20%20%20%20%20%20Shows%20this%20screen.%5Cn%20%20%20%20--min_count%3D%3Ckn%3E%20%20%20%20%20Minimal%20number%20of%20reads%20in%20a%20pileup%20line%20in%20order%20to%20consider%20it%20in%20the%20analysis%20%5Bdefault%3A%200%5D%5Cn%20%20%20%20--gene_regions%3D%3Ckn%3E...%20%20All%20the%20gene%20regions%20you%20want%20to%20plot.%20opts%3A%205utr%2C%203utr%2C%20exon.%20The%20default%20is%20all%20of%20them.%20Separate%20them%20by%20space%20%5Bdefault%3A%205utr%203utr%20exon%5D&help=on&replace=on&stdopt=on) with this feature, but not widly used and verified like `docopt`
